src/style.css

```
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block * {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  color: white;
+  height: 300px;
+  min-width: 300px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+  flex-basis: 33%;
+}
+
+.block1 {
+  background: rgb(255,0,0);
+}
+
+.block2 {
+  background: rgb(calc(255 * 0.8),0,0);
```

you can use percentage here. And use background-color instead of background shorthand.
https://developer.mozilla.org/en-US/docs/Web/CSS/background
```suggestion
  background-color: rgb(80%, 0%, 0%);
```